spec/javascripts/app/models/person_spec.js

```diff
@@ -1,69 +1,69 @@
 
 describe("app.models.Person", function() {
   beforeEach(function() {
-    this.mutual_contact = factory.person({relationship: 'mutual'});
-    this.sharing_contact = factory.person({relationship :'sharing'});
-    this.receiving_contact = factory.person({relationship: 'receiving'});
-    this.blocked_contact = factory.person({relationship: 'blocked', block: {id: 1}});
+    this.mutualContact = factory.person({relationship: "mutual"});
+    this.sharingContact = factory.person({relationship: "sharing'});
+    this.receivingContact = factory.person({relationship: "receiving"});
```

Comment:
Expected '}' to match '{' from line 5 and instead saw 'receiving'.<br>Unclosed string.